motion_tracker/utils/crop_vis.py

```diff
@@ -1,7 +1,7 @@
 import cv2
-from motion_tracker.utils.image_cropping import Coords, crop_and_resize
+from image_cropping import Coords, crop_and_resize
 
```

Comment:
So, watch out for this if you try to run this stuff. For some reason this wasn't working for me even when running from the main directory of the repo. Not sure if I don't have something setup that you do, or what. But, I had to revert to that import for it to work.